### ISSUE TITLE
Expose MUCLight change affiliation REST API

### ISSUE THREAD

**Issue by jasl** [esl/MongooseIM#2333]
MongooseIM version: 3.3.0
Installed from: source

It seems REST API doesn't have ability of manage MUCLight members' affiliation.

But when I discover source code, I found <https://github.com/esl/MongooseIM/blob/master/src/muc_light/mod_muc_light_commands.erl#L176>, it looks like for change affiliation, but didn't expose yet.

**Comment by fenek** [comment]
Good catch, indeed it looks like it was skipped by mistake. I've added an internal ticket to fix this.

**Comment by JanuszJakubiec** [comment]
We are no longer extending the REST API, because it is deprecated. The functionality is covered in new GraphQL API and it contains commands like kickUser and inviteUser that covers the functionality.
URL to the GraphQL documentation: https://esl.github.io/MongooseDocs/latest/graphql-api/admin-graphql-doc.html#definition-MUCLightAdminMutation
